### PR TITLE
cookiecutter v2.5.0

### PR DESCRIPTION
**Pull request by lorepirri**
[PKG-3351] cookiecutter 2.5.0

dev url: https://github.com/cookiecutter/cookiecutter/tree/2.5.0
conda-forge: https://github.com/conda-forge/cookiecutter-feedstock/blob/main/recipe/meta.yaml
dependencies: https://github.com/cookiecutter/cookiecutter/blob/2.5.0/setup.py
pypi: https://pypi.org/project/cookiecutter/

Destination channel: **defaults**

**Changes**
- version 2.5.0 (latest)

**Dependency for**
- [kedro v0.18.14 ❄️](https://anaconda.atlassian.net/browse/PKG-3345)

[PKG-3351]: https://anaconda.atlassian.net/browse/PKG-3351?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -23,47 +23,56 @@ requirements:
...
-    - python 3.*
+    - python
   source_files:
     - tests
   commands:
-    - python -m pip check
+    - pip check
     - cookiecutter --version
+    - cookiecutter --version | grep -iE "Cookiecutter {{ version.replace(".", "\\.") }}"
+    - pip list
```
Comment (boldorider4) [inline recipe/meta.yaml:57]: `pip list` can be removed because of next line.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -23,47 +23,56 @@ requirements:
...
-    - pytest-cov
     - pytest-mock
-    - python 3.*
+    - python
   source_files:
     - tests
   commands:
-    - python -m pip check
+    - pip check
     - cookiecutter --version
```
Comment (boldorider4) [inline recipe/meta.yaml:55]: this line can be replaced with the next one.

**Review by boldorider4 — APPROVED** [review]
LGTM, only a couple of remarks.

File: recipe/meta.yaml (hunk 2 above)
Comment (lorepirri) [reply recipe/meta.yaml]: ok

File: recipe/meta.yaml (hunk 1 above)
Comment (lorepirri) [reply recipe/meta.yaml]: ok

**Review by cbouss — APPROVED** [review]

**Pull request merged**